readthedocs-theme/templates/includes/topnav.html

```diff
@@ -1,14 +1,14 @@
 <div class="ui menu fixed borderless" id="topnav">
   <div class="ui container">
 
-    <a class="header item" href="">
-      <div class="ui image readthedocs-logo"><img src="/theme/img/logo-wordmark-dark.svg"/></div>
+    <a class="header item" href="{{ SITEURL }}/">
+      <div class="ui image readthedocs-logo"><img src="{{ SITEURL }}/theme/img/logo-wordmark-dark.svg"/></div>
     </a>
 
-    <a class="item active" href="">Product</a>
-    <a class="item" href="">Pricing</a>
-    <a class="item" href="">Company</a>
-    <a class="item" href="">Resources</a>
+    <a class="item{% if page and page.slug == "product" %} active{% endif %}" href="{{ SITEURL }}/product.html">Product{% if page and page.slug == "product" %} <span class="sr-only">(current)</span>{% endif %}</a>
```

Comment:
What's `sr-only`? Looks like a custom class to help screen readers? I would stick with ARIA attributes here instead probably.

Reply:
It's fairly common to add screen-reader only content with this approach -- see https://tailwindcss.com/docs/screen-readers#screen-reader-only-elements for example (Tailwind wasn't the first to do this, I think Bootstrap was).

TIL about aria-current!

Reply:
Ahh likewise, TIL about `sr-only`. That does seem useful for some cases where supplementing graphic/visual content is maybe not a good fit for ARIA attributes alone.

I don't have enough experience using assistive technologies to say which one is better for users, but my guess would be that `aria-current` or similar would give more context to AT than just the hidden string content would. Overall, I probably lean towards ARIA as it's a supported standard, but it is handy to have `sr-only` and friends for edge cases.

Reply:
I've learned a bit more about [WCAG](https://www.w3.org/TR/wai-aria-1.1/#aria-current) today too! `aria-current="page"` seems to be the better option here.